x-pack/test/alerting_api_integration/common/fixtures/plugins/actions_simulators/server/README.md

```diff
@@ -17,7 +17,7 @@ simulator usage
 
 This may get out of date, consult the code for exact urls and inputs.  Each
 simulator's last path segment should be the name of the service (eg, slack,
-pagerduty, etc).
+pagerduty, swimlane, etc).
```

Comment:
As other service are not listed I think this should be decided by the alerting team

Reply:
We welcome more simulators!  Thanks!

At some point (there is an issue open somewhere), we'd like to add tests with real services, but that's going to be a bit of work, getting the services set up, secrets in a vault, etc.